application/src/test/java/eu/europa/ec/fisheries/uvms/activity/service/util/MapperUtil.java

```diff
@@ -518,7 +518,7 @@ public static FLUXLocation getFluxLocation() {
         return fluxLocation;
     }
 
-    public static FLUXReportDocument getFluxReportDocument() {
+    public static FLUXReportDocument getFluxReportDocumentLowerCase() {
```

Comment:
Is it possible to get even better method names? These are not identical FLUXReportDocuments, this one is a correction (I think, CodeType "5") and the other is a new report (CodeType "9").

Reply:
Good idea! I just saw that the cases of the Id was different. New/Correction is much better